modules/virt-importing-rhel-image-boot-source-web.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-and-using-boot-sources.adoc
+
+[id="virt-importing-rhel-image-boot-source-web_{context}"]
+= Importing a Red Hat Enterprise Linux image as a boot source
+
+You can import a Red Hat Enterprise Linux (RHEL) image as a boot source by specifying the URL address for the image.
+
+.Prerequisites
+* To import a RHEL image via a URL, access to the web server with the operating system image is required. For example: the Red Hat Enterprise Linux web page with images.
+
+.Procedure
+
+. In the {VirtProductName} console, click *Workloads* -> *Virtualization* from the side menu.
+
+. Click the *Templates* tab.
+
+. Identify the RHEL template that you want to use for creating a virtual machine.
```

Comment:
Since this procedure is for adding a boot source, I would suggest changing this to "Identify the RHEL template for which you want to configure a boot source." 
You could also combine steps 3 and 4 by adding "and click Add source" to my suggested sentence.

Reply:
Updated the content.